app/helpers/pivotal_helper.rb

```diff
@@ -0,0 +1,6 @@
+module PivotalHelper
+  def form_button
```

Comment:
can we name this differently? `form_button` sounds like it renders a generic button tag, maybe `token_form_button_text`?